local/generate-keys.sh

```diff
@@ -89,5 +91,35 @@ rm ./miw/keycloak-setup-temp.json
 
 SUPPLIER_CERT_SHA="$(openssl x509 -in "$SUPPLIER_CERT" -noout -sha256 -fingerprint | tr '[:upper:]' '[:lower:]' | tr -d : | sed 's/.*=//')"
 
+echo "Creating key pair for mock iam"
```

Comment:
Lines 70, 87 and 90 could be commented out, as long as the MIW is not in use.

Reply:
Just for explanation: This is a key pair used for the decentralized identities (DID) in the iam-mock. 

The public key is published with the did so others can verify. The private key is used to sign the JWT. Actually as it's a mock I use one pair for all :)